backend/selenium_tests/people/test_people.py

```diff
@@ -26,7 +33,7 @@ def social_worker_program() -> Program:
 
 
 @pytest.fixture
-def add_people(social_worker_program: Program) -> None:
+def add_people(social_worker_program: Program) -> []:
```

Comment:
https://docs.pytest.org/en/7.1.x/how-to/fixtures.html

how about we yield instead of return from that fixture
also, `-> []` will not be the case here.